src/pages/Selector/index.jsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import container from './container';
 
+/* eslint-disable */
```

Comment:
does eslint need disabled here? is it maybe only the container?

Reply:
It is throwing up the error in both places. But looking at it I think it I will add in a comment showing that this is temporary.